game-app/game-core/src/test/java/games/strategy/triplea/ui/CasualtySelectionTest.java

```diff
@@ -0,0 +1,94 @@
+package games.strategy.triplea.ui;
+
+import static games.strategy.triplea.Constants.UNIT_ATTACHMENT_NAME;
+import static games.strategy.triplea.ui.CasualtySelection.playerMayChooseToDistributeHitsToUnitsWithDifferentMovement;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.GamePlayer;
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.data.UnitType;
+import games.strategy.triplea.attachments.UnitAttachment;
+import games.strategy.triplea.xml.TestMapGameData;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class CasualtySelectionTest {
+  CasualtySelectionTest() {
+    System.out.println("costrictiong CasualtySelectionTest");
```

Comment:
Is this a stray-debug statement?

Reply:
I wasn't completely tidy in the test code, sorry. I will push an update.